examples/htlc-pyteal-ts/scripts/withdraw/htlc-withdraw.ts

```diff
@@ -6,35 +6,43 @@
  * the TEAL program. The TEAL program will transfer its balance to the seller
  * if the seller is able to provide the secret value that corresponds to the hash in the program.
 */
-const { TransactionType, SignType, stringToBytes } = require('@algorand-builder/algob');
-const { executeTransaction, prepareParameters } = require('./common');
+import {
+  AlgobDeployer,
+  AlgobRuntimeEnv,
+  AlgoTransferParam,
+  SignType,
+  stringToBytes,
+  TransactionType
+} from "@algorand-builder/algob";
 
-async function run (runtimeEnv, deployer) {
+import { executeTx, prepareParameters } from "./common";
+
+async function run (runtimeEnv: AlgobRuntimeEnv, deployer: AlgobDeployer): Promise<void> {
   const { alice, scTmplParams, secret } = prepareParameters(deployer);
   const wrongSecret = 'hero wisdom red split loop element vote belt';
 
   let lsig = await deployer.loadLogic('htlc.py', [stringToBytes(wrongSecret)], scTmplParams);
   let sender = lsig.address();
 
-  const txnParams = {
+  const txnParams: AlgoTransferParam = {
     type: TransactionType.TransferAlgo,
     sign: SignType.LogicSignature,
-    fromAccount: { addr: sender },
+    fromAccount: { addr: sender, sk: new Uint8Array(0) },
```

Comment:
please add a comment here, `we don't need secret key for logic signature account` or remove sk part.

Reply:
We can't remove sk part, as it will give a type error now (`type Account` requires both `addr` & `sk`). Adding comment.